vernac/declareUniv.ml

```diff
@@ -157,6 +211,33 @@ let do_universe ~poly l =
     in
     Global.push_section_context ctx
 
+    (* TODO: move to its proper place *)
```

Comment:
what does this mean?

Reply:
When I first wrote the function I was wondering whether this should go in its own file `declareSort.ml` or something similar. Do you think that would be better ?

Reply:
It should be fine here imo